src/goto-programs/goto_contractor.h

```diff
@@ -18,10 +18,9 @@
 #include <ibex/ibex_Ctc.h>
 #include <irep2/irep2.h>
 #include <util/type_byte_size.h>
-#include <goto-programs/abstract-interpretation/interval_domain.h>
 #include <goto-programs/abstract-interpretation/interval_analysis.h>
 #include <limits>
-
+#include <chrono>
```

Comment:
I don't see \<chrono\> being used here.

Reply:
removed it

Reply:
Hm, apparently there is: `/esbmc/src/goto-programs/goto_contractor.h:637:21: error: ‘std::chrono’ has not been declared
  637 |     auto t_0 = std::chrono::steady_clock::now();``

I am double checking which change actually caused the chrono to be an issue now

Reply:
This PR changed the includes of the goto_contractor.h so it does no longer include the interval_domain. I guess some part of that chain would also include the chrono